src/test/java/ch/uzh/ifi/hase/soprafs22/screwyourneighborserver/api/CardApiTest.java

```diff
@@ -157,4 +159,88 @@ void hide_correct_cards() {
         .jsonPath("_embedded.matches[*].rounds[0].cards[0].cardSuit")
         .isEqualTo(CLUB.name());
   }
+
+  @Test
+  void accept_card_playing() {
+
+    HttpHeaders responseHeaders =
+        webTestClient
+            .post()
+            .uri("/players")
+            .body(BodyInserters.fromValue(PLAYER_1))
+            .exchange()
+            .expectStatus()
+            .isCreated()
+            .expectBody()
+            .returnResult()
+            .getResponseHeaders();
+
+    String sessionIdPlayer1 = getSessionIdOf(responseHeaders);
+
+    responseHeaders =
+        webTestClient
+            .post()
+            .uri("/players")
+            .body(BodyInserters.fromValue(PLAYER_2))
+            .exchange()
+            .expectStatus()
+            .isCreated()
+            .expectBody()
+            .returnResult()
+            .getResponseHeaders();
+
+    String sessionIdPlayer2 = getSessionIdOf(responseHeaders);
+
+    Player player1 =
+        playerRepository.findAll().stream()
+            .filter(player -> player.getName().equals(PLAYER_NAME_1))
+            .findFirst()
+            .orElseThrow();
+
+    Player player2 =
+        playerRepository.findAll().stream()
+            .filter(player -> player.getName().equals(PLAYER_NAME_2))
+            .findFirst()
+            .orElseThrow();
+
+    Game game =
+        GameBuilder.builder("game1", gameRepository, participationRepository, playerRepository)
+            .withParticipationWith(player1)
+            .withParticipationWith(player2)
+            .withGameState(GameState.PLAYING)
+            .withMatch()
+            .withMatchState(MatchState.PLAYING)
+            .withHandForPlayer(PLAYER_NAME_1)
+            .withCards(ACE_OF_CLUBS, QUEEN_OF_CLUBS)
+            .finishHand()
+            .withRound()
+            .finishRound()
+            .finishMatch()
+            .build();
+
+    gameRepository.saveAll(List.of(game));
+    Card card =
+        cardRepository.findAll().stream()
+            .filter(c -> c.getRound() == null)
+            .filter(c -> c.getCardRank().equals(CardRank.ACE))
+            .findAny()
+            .orElseThrow();
+    Round round = game.getLastMatch().orElseThrow().getLastRound().orElseThrow();
+    card.setRound(round);
+
+    String uri = "cards/" + card.getId().toString() + "/round";
```

Comment:
Patching over these relation paths (cards/{id}/round is a little silly and i don't know how that would work.
I would stick to the normal patch operations on entities. (without additional suffixes)

Reply:
Hmm, ok, I don't get that. Maybe you have time to explain that some time.